src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Runtime.cs

```diff
@@ -401,6 +406,130 @@ private static DateTime CreateDateTime(double ticks)
             return unixTime.DateTime;
         }
 
+        private static object? ObjectToEnum(IntPtr methodHandle, int parm, object obj)
```

Comment:
Cannot we pass parameter type directly?

Reply:
Not sure will need to check but do not think we have it.